kernel/aster-nix/src/fs/pipe.rs

```diff
@@ -18,6 +18,8 @@ use crate::{
     time::clocks::RealTimeCoarseClock,
 };
 
+pub const PIPE_BUF_SIZE: usize = 65536;
```

Comment:
We already have a `PIPE_BUF_SIZE` in `kernel/aster-nix/src/syscall/pipe.rs`, I think we should have only one const unless named/anonymous pipe use a different internal buffer size? This should be checked in our baseline Linux. If they all are `65536`, we can unify them in current location `kernel/aster-nix/src/fs/pipe.rs`, plus name `DEFAULT_PIPE_BUF_SIZE` or `DEFAULT_PIPE_SIZE` is better I suppose.